packages/matter-node.js/src/Controller.ts

```diff
@@ -22,25 +22,41 @@ import { UdpInterface } from "./net/UdpInterface";
 import { getIntParameter, getParameter } from "./util/CommandLine";
 import { MdnsScanner } from "./matter/mdns/MdnsScanner";
 import { Logger } from "./log/Logger";
+import { StorageNode } from "./persistence/StorageNode";
+import { PersistenceManager } from "./persistence/PersistenceManager";
 
 const logger = Logger.get("Controller");
 
+const storage = new StorageNode(getParameter("file") || "controller.json");
+
 class Controller {
     async start() {
         logger.info(`node-matter`);
 
+        const persistenceManager = new PersistenceManager(storage);
+        await persistenceManager.initialize();
+
         const ip = getParameter("ip");
         if (ip === undefined) throw new Error("Please specify the IP of the device to commission with -ip");
         const port = getIntParameter("port") ?? 5540;
         const discriminator = getIntParameter("discriminator") ?? 3840;
         const setupPin = getIntParameter("pin") ?? 20202021;
-        const client = await MatterController.create(await MdnsScanner.create(), await UdpInterface.create(5540, "udp4"), await UdpInterface.create(5540, "udp6"));
+        const client = await MatterController.create(await MdnsScanner.create(), await UdpInterface.create(5540, "udp4"), await UdpInterface.create(5540, "udp6"), persistenceManager);
```

Comment:
Why does the network interface need the persistenceManager?

Reply:
Also why do we hard-code the port here?  The client port should be ephemeral and different from the default server port (which is okay to default to 5540, but should be defined as a const minimally rather than a magic number).

Reply:
1.) Not the Network interface needs the persistenceManager but the "MatterController" class. thats why it is added here for the Controller builder. I think this all will be changed with teh Hight level API completely

2.) In one of my other PRs the port will be configurable. It was hardcoded until now in node-matter